index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <title>To do list</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="container__title">
```

Comment:
div not needed, you can use grid in the `container` to put the title in the right place

Reply:
The div help to mix the background with the container__content.